README.md

```diff
@@ -12,41 +12,41 @@ Documentation
 -------------
 
 Documentation for Puppet and related projects can be found online at the
-[Puppet Docs site](https://docs.puppetlabs.com).
+[Puppet Docs site](https://puppet.com/docs).
 
 HTTP API
 --------
-[HTTP API Index](https://docs.puppetlabs.com/puppet/latest/reference/http_api/http_api_index.html)
+[HTTP API Index](https://puppet.com/docs/puppet/4.10/http_api/http_api_index.html)
 
 Installation
 ------------
 
-The best way to run Puppet is with [Puppet Enterprise](https://puppetlabs.com/puppet/puppet-enterprise),
+The best way to run Puppet is with [Puppet Enterprise](https://puppet.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://docs.puppetlabs.com/pe/latest)
+[The PE documentation is available here.](https://puppet.com/docs/pe/latest)
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html)
+[see the installation guide on the docs site.](http://puppet.com/docs/puppet/5.3/install_pre.html)
 
 If you need to run Puppet from source as a tester or developer,
-[see the running from source guide on the docs site.](https://docs.puppetlabs.com/guides/from_source.html)
+[see the running from source guide on the docs site.](https://docs.puppet.com/puppet/3.8/from_source.html)
```

Comment:
Following this link leads to something that mentions puppet *enterprise* 3.8, which is both confusing (the section before says "puppet"?) and seems old.

Reply:
I think the docs site is messed up here, somehow. It says it refers to enterprise 3.8, but the sidebar says the version refers to Puppet (and choosing some of the other versions leads to 404s)

Reply:
It is old. There hasn't been a build-from-source doc since Puppet 3.8. PUP-8200 in the ticket for that. There's also a note in the previous PR review by @joshcooper: https://github.com/puppetlabs/puppet/pull/6539#pullrequestreview-89319465

Reply:
Also, even if I don't update that link, https://docs.puppetlabs.com/guides/from_source.html redirects to the same URL anyway. The change is mostly in case the docs.puppetlabs.com domain is retired.

Reply:
FWIW the old URL https://docs.puppet.com/puppet/guides/from_source.html redirects to the new one... So it's... terrible